website/content/docs/getting-started.md

```diff
@@ -39,14 +39,14 @@ Heron client installer
 Uncompressing......
 Heron is now installed!
 
-Make sure you have "/usr/local/bin" in your path.
+Make sure you have "~/bin" in your path.
```

Comment:
Great point, done.  I searched other website/script locations requiring export and found it in Dockerfile, will update in future PR.